cupsfilters/image-jpeg-xl.c

```diff
@@ -166,57 +155,56 @@ _cfImageReadJPEGXL(cf_image_t *img, FILE *fp,
   
   /* Determine file size */
   if (fseek(fp, 0, SEEK_END) != 0) {
-    fprintf(stderr, "JPEG‑XL: fseek failed.\n");
+    DEBUG_printf(("JPEG‑XL: fseek failed.\n"));
     return -1;
   }
   filesize = ftell(fp);
+  DEBUG_printf(("DEBUG: File size: %zu bytes\n", filesize));
   rewind(fp);
-  fprintf(stderr, "DEBUG: File size: %zu bytes\n", filesize);
 
   data = malloc(filesize);
   if (!data) {
-    fprintf(stderr, "JPEG‑XL: Memory allocation failed.\n");
+    DEBUG_printf(("JPEG‑XL: Memory allocation failed.\n"));
     return -1;
   }
   bytesRead = fread(data, 1, filesize, fp);
+  DEBUG_printf(("DEBUG: Bytes read: %zu bytes\n", bytesRead));
   if (bytesRead != filesize) {
-    fprintf(stderr, "JPEG‑XL: File read error (read %zu of %zu bytes).\n", bytesRead, filesize);
+    DEBUG_printf(("JPEG‑XL: File read error. Expected %zu bytes, got %zu bytes\n", filesize, bytesRead));
     free(data);
     return -1;
   }
 
   /* Create the libjxl decoder */
   decoder = JxlDecoderCreate(NULL);
   if (!decoder) {
-    fprintf(stderr, "JPEG‑XL: Failed to create decoder.\n");
+    DEBUG_printf(("JPEG‑XL: Failed to create decoder.\n"));
     free(data);
     return -1;
   }
   
   status = JxlDecoderSetInput(decoder, data, filesize);
   if (status != JXL_DEC_SUCCESS) {
-    fprintf(stderr, "JPEG‑XL: Failed to set input buffer (status=%d).\n", status);
+    DEBUG_printf(("JPEG‑XL: Failed to set input buffer. Status: %d\n", status));
     JxlDecoderDestroy(decoder);
     free(data);
     return -1;
   }
-  fprintf(stderr, "DEBUG: Input buffer set successfully.\n");
 
-  // Process input until full image is decoded.
+  /* Process input until full image is decoded. */
```

Comment:
For comments please use `// ...` and not `/* ... */`  to align with the other files of libcupsfilters (and with OpenPrinting's code style).